site/_scss/blocks/_screenshot.scss

```diff
@@ -0,0 +1,15 @@
+.screenshot {
+  border: 1px solid var(--color-hairline);
+  // This ensures images are scaled to the exact size specified by the
+  // width attribute, preventing HiDPI screenshots from being blurry:
+  //  <img src="800x500.png" width="400" class="screenshot">
+  box-sizing: content-box;
+  padding: get-size(400);
```

Comment:
I think this is more padding than you want. On web.dev we only use 4px of padding but this will do 24px.